src/lib/openapi/spec/group-schema.ts

```diff
@@ -24,6 +24,10 @@ export const groupSchema = {
                 type: 'string',
             },
         },
+        rootRole: {
+            type: 'number',
+            nullable: true,
+        },
```

Comment:
Boyscout: Adding description to the new field?

Reply:
Good call! Done!